client/src/components/Button/Button.css

```diff
@@ -1,3 +1,15 @@
+/* button {
+	background-color: white;
+
+} */
 button {
```

Comment:
Please use a specific class name when using CSS. If not this style will affect all button in the project.